charts/orchestrator/templates/rhdh-operator.yaml

```diff
@@ -0,0 +1,215 @@
+{{- if and (eq $unmanagedNamespaceExists "false") .Values.rhdhOperator.enabled }}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: {{ .Values.rhdhOperator.subscription.namespace }}
+---
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: rhdh-operator-group
+  namespace: {{ .Values.rhdhOperator.subscription.namespace }}
+spec: {}
+---
+{{- end }}
+{{- if ne .Values.rhdhOperator.subscription.sourceImage "" }}
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: {{ .Values.rhdhOperator.subscription.source }}
+  namespace: openshift-marketplace
+spec:
+  displayName: RHDH Operator (Development)
+  image: {{ .Values.rhdhOperator.subscription.sourceImage }}
+  sourceType: grpc
+---
+{{- end }}
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: rhdh-operator
+  namespace: {{ .Values.rhdhOperator.subscription.namespace }}
+spec:
+  channel: {{ .Values.rhdhOperator.subscription.channel }}
+  installPlanApproval: Automatic
+  name: {{ .Values.rhdhOperator.subscription.pkgName }}
+  source: {{ .Values.rhdhOperator.subscription.source }}
+  sourceNamespace: {{ .Values.rhdhOperator.subscription.sourceNamespace | default (include "get-default-catalogsource-namespace" .) | quote }}
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: backstage-backend-auth-secret
+  namespace: {{ .Values.rhdhOperator.subscription.namespace }}
+stringData:
+  BACKEND_SECRET: {{ randAlphaNum 24 | nospace | b64enc }}
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: app-config-rhdh
+  namespace: {{ .Values.rhdhOperator.subscription.namespace }}
+data:
+  "app-config-rhdh.yaml": |
+    app:
+      title: Red Hat Developer Hub
+      baseUrl: https://backstage-backstage-{{ .Values.rhdhOperator.subscription.namespace }}.{{ include "cluster.domain" . }}
+    backend:
+      auth:
+        keys:
+          - secret: "${BACKEND_SECRET}"
+      baseUrl: https://backstage-backstage-{{ .Values.rhdhOperator.subscription.namespace }}.{{ include "cluster.domain" . }}
+      csp:
+        frame-src:
+          - "https://sandbox.kie.org"
+      cors:
+        origin: https://backstage-backstage-{{ .Values.rhdhOperator.subscription.namespace }}.{{ include "cluster.domain" . }}
+      database:
+        client: pg
+        connection:
+          password: ${POSTGRESQL_ADMIN_PASSWORD}
+          user: ${POSTGRES_USER}
+          host: ${POSTGRES_HOST}
+          port: ${POSTGRES_PORT}
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: dynamic-plugins-rhdh
+  namespace: {{ .Values.rhdhOperator.subscription.namespace }}
+data:
+  "dynamic-plugins.yaml": |
+    includes:
+      - dynamic-plugins.default.yaml
+    plugins:
+      - disabled: false
+        package: "@janus-idp/backstage-plugin-orchestrator-backend-dynamic@1.3.1"
+        integrity: >-
+          sha512-kDbhP8Cq3PJFLJ6bwTRJwllxU71bnH7SkC3I3HqX6K9TVIqSKwyI8MNGYMRHoTlxwHH+4fASmFBPfEtIpMVLTA==
```

Comment:
maybe worth to document in the README the changes needed to integrate a new version of RHDH operator?

Reply:
Thanks for the advice. added a section for it.